### ISSUE TITLE
User Streaming and POST direct_messages/new APIs are now Deprecated. Needs an update.

### ISSUE THREAD

**Issue by kumarabhirup** [KumarAbhirup/glitter#4]
Instead of  streaming `GET user`, `account activity` is now recommended by Twitter. And instead of using `POST direct_messages/new` twitter has new endpoint of `POST direct_messages/events/new`.  By August 2018, streamers and old DM endpoints will not work. If not changed, whole structure will crash.

**Comment by ghost** [comment]
sad

**Comment by kumarabhirup** [comment]
Yes, it is. Twitter made their APP approval process very strict. 😒
